products/cakephp.md

```diff
@@ -0,0 +1,81 @@
+---
+title: CakePHP
+permalink: /cakephp
+alternate_urls:
+  - /cake-php
+  - /cake
+category: framework
+iconSlug: cakephp
+releasePolicyLink: https://book.cakephp.org/4/en/release-policy.html
+releaseLabel: "CakePHP __RELEASE_CYCLE__ __CODENAME__"
+releaseColumn: true
+releaseDateColumn: true
+activeSupportColumn: true
+discontinuedColumn: false
+versionCommand: php bin/cake.php version
+
+auto:
+  - git: https://github.com/cakephp/cakephp.git
+
+releases:
+  - releaseCycle: "4.4"
+    eol: false
+    support: true
+    releaseDate: 2019-12-15
+    latest: "4.4.5"
+    latestReleaseDate: 2022-08-28
+    link: https://bakery.cakephp.org/2019/12/15/cakephp_400_released.html
```

Comment:
we use links to latest tag's release links

Reply:
that was the reason i commited this one : https://github.com/endoflife-date/endoflife.date/pull/1672/commits/d596e1d3535aa369372d854716580f9d380ae946

Reply:
That wasn't clear form the template. Fixed that in https://github.com/endoflife-date/endoflife.date/pull/1674

Reply:
I mistakenly reverted it because of an out of date version of teh file

Reply:
i know it is completely ok :)

Reply:
Re-done in https://github.com/endoflife-date/endoflife.date/pull/1672/commits/041dd679b4acbfaaacfb9d95bebf2c942c0375ea